src/main/java/com/minecolonies/coremod/client/gui/WindowHutGuardTower.java

```diff
@@ -216,7 +219,7 @@ private void switchTask(final Button button)
     {
         if (button.getID().contains("patrol"))
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'BUTTON_PATROL_MODE' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1192)